src/app.js

```diff
@@ -557,6 +572,18 @@ var run = function () {
         files.addReadOnly(url, content)
         cb(null, content)
       })
+    } else if ((match = /^(ipfs:\/\/?.+)/.exec(url))) {
+      $('#output').append($('<div/>').append($('<pre/>').text('Loading ' + url + ' ...')))
+      handleIPFS(match[1], function (err, content) {
+        if (err) {
+          cb('Unable to import "' + url + '": ' + err)
```

Comment:
Please to it in this pr, if possible. It's not a complicated change and three times the same code is already two times too much :-)